### ISSUE TITLE
Tutorial 01 randomly fails in CI

### ISSUE THREAD

**Issue by jngrad** [espressomd/espresso#3928]
The RDF check `atol=0.1` is too strict, failed with 0.10175872 ([logfile](https://gitlab.icp.uni-stuttgart.de/espressomd/espresso/-/jobs/255261)).

```
 3/25 Test  #3: tutorial_01-lennard_jones ......................................***Failed    4.03 sec
.F
======================================================================
FAIL: test_rdf (__main__.Tutorial)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builds/espressomd/espresso/build/testsuite/scripts/tutorials/test_01-lennard_jones.py", line 30, in test_rdf
    np.testing.assert_allclose(tutorial.rdf, tutorial.theo_rdf,
  File "/usr/lib/python3/dist-packages/numpy/testing/_private/utils.py", line 1514, in assert_allclose
    assert_array_compare(compare, actual, desired, err_msg=str(err_msg),
  File "/usr/lib/python3/dist-packages/numpy/testing/_private/utils.py", line 841, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Not equal to tolerance rtol=0, atol=0.1
Mismatch: 1%
Max absolute difference: 0.10175872
Max relative difference: 1.
 x: array([0.      , 0.      , 0.      , 0.      , 0.      , 0.      ,
       0.      , 0.      , 0.      , 0.      , 0.      , 0.      ,
       0.      , 0.      , 0.      , 0.      , 0.      , 0.      ,...
 y: array([0.000000e+000, 0.000000e+000, 0.000000e+000, 0.000000e+000,
       0.000000e+000, 0.000000e+000, 0.000000e+000, 0.000000e+000,
       0.000000e+000, 0.000000e+000, 0.000000e+000, 0.000000e+000,...
----------------------------------------------------------------------
Ran 2 tests in 0.005s
FAILED (failures=1)
```

@schlaicha

**Comment by jngrad** [comment]
I typically run stochastic tutorial tests in a loop to make sure the checks are not too strict (third code block in [wiki/Testing](https://github.com/espressomd/espresso/wiki/Testing#writing-tests-2)).

**Comment by jngrad** [comment]
Using 46fd5f133a78c9e, 75 runs out of 1007 resulted in an RDF with `atol>0.1`:
![histogram of the RDF check](https://user-images.githubusercontent.com/42201407/94960895-ec1fc080-04f3-11eb-8962-e236989ccf11.png)

```sh
test_type="tutorials"
test_name="01-lennard_jones"
make local_${test_type}
cd testsuite/scripts/${test_type}/
for i in {1..1000}; do
  rm -f local_${test_type/tutorials/${test_type}/${test_name%-[0-9]}}/${test_name}_processed.py
  ../../../pypresso test_${test_name}.py | tee -a tutorials.log
done
```

**Comment by schlaicha** [comment]
I briefly discussed today offline with @RudolfWeeber , I will add add a seed to the numpy.random calls which removes the stochastic nature of the test.

**Comment by jngrad** [comment]
Fixed by 7909555dd.
